__tests__/services/usersServices/getSingleUser.test.js

```diff
@@ -0,0 +1,41 @@
+const User = require("../../../models/user");
+const { getSingleUser } = require("../../../api/users/services");
+
+jest.mock("../../../models/user");
+
+const id = "123";
+const newId = "12345";
+
+describe("getSingleUser function", () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  it("should return user, when user exists", async () => {
+    const mockReturn = {
+      _id: id,
+      isDeleted: false
+    };
+
+    User.findOne.mockResolvedValueOnce(() => mockReturn);
+    const result = await getSingleUser(id);
+
+    expect(User.findOne).toHaveBeenCalledWith(mockReturn);
+    expect(result()).toBe(mockReturn);
```

Comment:
result() ??? Why `getSingleUser` returns method?

Probably you need to `User.findOne.mockResolvedValueOnce(mockReturn);` in line 20

Reply:
The same for the test below

Reply:
done